packages/common/src/utils/github.ts

```diff
@@ -1,5 +1,5 @@
 export const isGithubToken = (value: string) =>
-    !!value.match(/^(github_pat_|ghp_)/);
+    !!value.match(/^(github_pat_|ghp_|ghs_)/);
```

Comment:
tokens generated with installation have a `ghs_` prefix . I don't think we need to change the error message, since this is not introduced by the user.